pkg/deploy/api/types.go

```diff
@@ -437,6 +437,8 @@ type DeploymentCondition struct {
 	Type DeploymentConditionType
 	// Status of the condition, one of True, False, Unknown.
 	Status kapi.ConditionStatus
+	// The last time this condition was updated.
+	LastUpdateTime unversioned.Time
```

Comment:
LastTransitionTime godoc already mentions when it is updated
https://github.com/openshift/origin/blob/b61cf87413a73268ff35fdeec0bade6b2e6eb61e/pkg/deploy/api/types.go#L440